emloop_tensorflow/models/conv.py

```diff
@@ -38,14 +38,14 @@ def compute_pool_amount(encoder_config: Sequence[str]):
 
 def cnn_encoder(x: tf.Tensor,
                 encoder_config: Sequence[str],
-                is_training: Optional[tf.Tensor]=None,
-                activation: Callable[[tf.Tensor], tf.Tensor]=tf.nn.relu,
-                conv_kwargs: Optional[dict]=None,
-                bn_kwargs: Optional[dict]=None,
-                ln_kwargs: Optional[dict]=None,
-                skip_connections: Optional[List[tf.Tensor]]=None,
-                use_bn: bool=False,
-                use_ln: bool=False) -> tf.Tensor:
+                is_training: Optional[tf.Tensor] = None,
```

Comment:
ugly for me... but whatever

Reply:
Its PEP8 bro..., even pycharm highlights it now